newIDE/app/src/BehaviorsEditor/Editors/BehaviorPropertiesEditor.js

```diff
@@ -0,0 +1,43 @@
+// @flow
+import * as React from 'react';
+import PropertiesEditor from '../../PropertiesEditor';
+import propertiesMapToSchema from '../../PropertiesEditor/PropertiesMapToSchema';
+import EmptyMessage from '../../UI/EmptyMessage';
+import { type BehaviorEditorProps } from './BehaviorEditorProps.flow';
+
+type Props = BehaviorEditorProps;
+
+const styles = {
+  propertiesContainer: {
+    padding: 10,
+  },
+};
+
+export default class BehaviorPropertiesEditor extends React.Component<Props> {
+  render() {
+    const { behavior, project } = this.props;
+    const properties = behavior.getProperties(project);
+
+    const propertiesSchema = propertiesMapToSchema(
+      properties,
+      behavior => behavior.getProperties(project),
+      (behavior, name, value) => behavior.updateProperty(name, value, project)
+    );
+
+    return (
+      <div style={styles.propertiesContainer}>
```

Comment:
I've copied this editor almost exactly for the future default editor for objects 👍
Note that we can use `<Column>` instead of the `div` and remove the custom style. This will use the margins from `Column` which are standardized across the app.

Reply:
Will change it then :)

Reply:
See https://github.com/4ian/GDevelop/pull/832/files#diff-ed36f79c6072b1dc0fcf031371c586c6R23 :)